ui/apps/platform/src/Containers/Dashboard/Widgets/ComplianceLevelsByStandard.tsx

```diff
@@ -72,27 +61,29 @@ function processData(
     if (!data) {
         return undefined;
     }
-    const { complianceStandards } = data;
-    const modifiedData = data.controls.results.map((result) => {
+    const { complianceStandards, controls } = data;
+
+    const modifiedData: ComplianceLevelByStandard[] = [];
+    controls.results.forEach((result) => {
         const aggregationName = result?.aggregationKeys[0]?.id;
         const standard = complianceStandards.find((cs) => cs.id === aggregationName);
-        const { numPassing, numFailing } = result;
-        const standardQueryValue =
-            standardLabels[standard?.id] || aggregationName || 'Unrecognized standard';
-        const query = getUrlQueryStringForSearchFilter({
-            ...searchFilter,
-            Cluster: searchFilter.Cluster || '*',
-            standard: standardQueryValue,
-        });
-        const link = `${complianceBasePath}/${
-            urlEntityListTypes[standardEntityTypes.CONTROL]
-        }?${query}`;
-        const modifiedResult = {
-            name: standard?.name || aggregationName || 'Unrecognized standard',
-            passing: Math.round((numPassing / (numFailing + numPassing)) * 100) || 0,
-            link,
-        };
-        return modifiedResult;
+
+        if (standard) {
```

Comment:
Good idea to opt for the `forEach` and ignore invalid entries, instead of the `map` approach which used fallback (and likely not useful) data.

Reply:
For your info, specific reason for `forEach` instead of `filter` and `map` is double `find` for standard.